### ISSUE TITLE
Resizable module browser

### ISSUE THREAD

**Issue by synnys** [VCVRack/Rack#1417]
I use a 34” ultra wide display (3440*1440) for rack and the module browser covers the whole screen so I can’t see the patch itself anymore. Despite the fact that I can see a whole lot of modules, I find it distracting. The choice for adding a new module is often based on what’s already in your rack. But if you can’t see that, that’s not an optimal workflow. I’d really love to be to be able to resize the browser and move/place it to the side of the rack for example. At least the option to do so.

**Comment by rubyglow** [comment]
Yeah, just from watching a couple of YT videos, I must say that the side-panel module browser of Voltage Modular seems like a very nice workflow. It should be collapsable/hideable of course.
